CHANGELOG-1.1.md

```diff
@@ -1,3 +1,32 @@
+# TiDB Operator v1.1.1 Release Notes
+
+## Notable changes
+
+- Add `additionalContainers` and `additionalVolumes` fields so that Tidb-operator can support adding sidecars to `TiDB`, `TiKV`, `PD`, etc. ([#2229](https://github.com/pingcap/tidb-operator/pull/2229), [@yeya24](https://github.com/yeya24))
+- Do not scale or upgrade TiKV at the same time ([#2705](https://github.com/pingcap/tidb-operator/pull/2705), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Fix the bug TidbMonitor will scrape multi TidbCluster with same same in different namespace when then namespace in `ClusterRef` not set ([#2746](https://github.com/pingcap/tidb-operator/pull/2746), [@Yisaer](https://github.com/Yisaer))
+- TiDB Operator examples updated to deploy TiDB Cluster 4.0.0 images ([#2756](https://github.com/pingcap/tidb-operator/pull/2756), [@ti-srebot](https://github.com/ti-srebot))
+- Add alertMangerAlertVersion option to TidbMonitor ([#2744](https://github.com/pingcap/tidb-operator/pull/2744), [@weekface](https://github.com/weekface))
+- Fix alert rules lost after rolling upgrade ([#2715](https://github.com/pingcap/tidb-operator/pull/2715), [@weekface](https://github.com/weekface))
+- Fix an issue that pods may be stuck in pending for a long time in scale-out after a scale-in ([#2709](https://github.com/pingcap/tidb-operator/pull/2709), [@cofyc](https://github.com/cofyc))
+- Add `EnableDashboardInternalProxy` in `PDSpec` to let user directly visit PD Dashboard ([#2713](https://github.com/pingcap/tidb-operator/pull/2713), [@Yisaer](https://github.com/Yisaer))
+- Fix the pv syncing error when `TidbMonitor` and `TidbCluster` have different value in `reclaimPolicy` ([#2707](https://github.com/pingcap/tidb-operator/pull/2707), [@Yisaer](https://github.com/Yisaer))
+- Update Configuration to v4.0.1 ([#2702](https://github.com/pingcap/tidb-operator/pull/2702), [@Yisaer](https://github.com/Yisaer))
+- Change tidb-discovery strategy type to `Recreate` ([#2701](https://github.com/pingcap/tidb-operator/pull/2701), [@weekface](https://github.com/weekface))
```

Comment:
I think the release note here should be `Fix the bug that discovery pod could exist one more instances in some cases`.

Reply:
cc @weekface what do you think?

Reply:
agree